docs/user_guides/getting_started.rst

```diff
@@ -103,6 +103,54 @@ To run the examples:
 
     python numba_dpex/examples/sum.py
 
+Docker
+------
+
```

Comment:
Need to add instruction/information about the other two packages: `builder`, `stages`.

Reply:
Done.